cmd/srcd/cmd/parse.go

```diff
@@ -79,6 +81,24 @@ The remaining nodes are printed to standard output in JSON format.`,
 			return err
 		}
 
+		if lang == "" {
+			lang, err = parseLang(ctx, c, path, b)
```

Comment:
I'm not sure we should use parse method of bblfsh file just to get the language of it.
I would better call enry here directly.

Or kept as it was before but check for [ErrMissingDriver](https://github.com/bblfsh/bblfshd/blob/master/daemon/daemon.go#L161) if bblfsh exports it. (if not, I think we can create an issue to bblfsh).

Reply:
Given that currently when the `--lang` flag is provided the language detection is skipped and performed otherwise, the idea is perform the language detection as a separate step in order to check whether the language is supported. Other than that I don't know which is the best solution for achieving this, and maybe `enry` (which I don't know) is actually the best option. I used `bblfsh` just because the logic was already present.

Reply:
I'm gonna check that `ErrMissingDriver` error.

Reply:
It seems that by simply checking whether the error thrown is of type `ErrMissingDriver` this PR could be simplified a lot. I'm just trying the understand [this error](https://github.com/grpc/grpc-go/issues/566) that I encountered when importing `bblfshd`.

Reply:
No need to import `bblfshd`. The error should be exported in [client-go](https://github.com/bblfsh/client-go). If it's not, I think we can request it. The use case for treating this error differently from just "internal error" is common in my opinion.

Reply:
AFAIU the only errors exported by [client-go](https://github.com/bblfsh/client-go) are [these](https://github.com/bblfsh/client-go/blob/master/request.go#L17). Actually I think that this shouldn't be an internal error too as you said. I'm gonna open the issue requesting this then.